physics/module_bl_mynn.F90

```diff
@@ -711,7 +688,7 @@ SUBROUTINE mynn_bl_driver(            &
           ENDDO
        ENDDO
 
-       IF ( bl_mynn_tkebudget ) THEN
+       IF (tke_budget .eq. 1) THEN
```

Comment:
Yes, this one is annoying. I had to change it to an integer because, in WRF, this flag is used as a package declaration (similar to how it's used in GFS_typedefs.F90, where 5 3D arrays are initialized when this is set to 1. However, WRF preprocessing can not handle logical flags, so I had to change it back to integer. Part of the many compromises for "universalization".

Reply:
Gotcha. I wasn't aware of the constraints from the WRF pp.